contracts/src/chainlink/solidity/emergency/StarknetValidator.sol

```diff
@@ -22,9 +22,13 @@ interface AggregatorValidatorInterface {
 /// @title StarknetValidator - makes cross chain calls to update the Sequencer Uptime Feed on L2
 contract StarknetValidator is TypeAndVersionInterface, AggregatorValidatorInterface, SimpleWriteAccessController {
   // Config for L1 -> L2 message cost approximation
+  // Calculated as k * gasEstimate * gasPriceL1Feed / 100
   struct GasConfig {
+    // gas units derived from starknet estimate_message_fee
     uint256 gasEstimate;
     address gasPriceL1Feed;
+    // scalar coefficient
+    uint32 k;
```

Comment:
Both the comment and the unit name aren't very descriptive. We used `gasAdjustment` on cosmos: https://github.com/smartcontractkit/chainlink-cosmos/blob/837a92027ab677d8b5a0168864c7fa03ef91a249/contracts/ocr2/src/contract.rs#L1498-L1500

And it's in terms of percent (e.g. 140 = scale the resulting estimate to 140% to give some buffer)

Reply:
cool, I'll update the names

Reply:
I'll consider making it more intuitive percent-wise

Reply:
Updated the comments. It should work exactly the same as cosmos. 100 = 100% of original value